### ISSUE TITLE
Whether Perspective Correction was neglected

### ISSUE THREAD

**Issue by No360201** [TEXTurePaper/TEXTurePaper#21]
https://github.com/TEXTurePaper/TEXTurePaper/assets/57511687/e49eba41-e3b4-497b-8932-423f546da18f

When we rotated the target, the texture was distorted, much like the result of not taking perspective correction into account，such as
![image](https://github.com/TEXTurePaper/TEXTurePaper/assets/57511687/11f61abe-8a9c-450a-a007-5eb0a24d070a)
@[eladrich](https://github.com/TEXTurePaper/TEXTurePaper/commits?author=eladrich)
eladrich

**Comment by Caenorst** [comment]
@No360201 you are correct in your assumption, we have [examples](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/gltf_viz.ipynb) in Kaolin using nvdiffrast that fixes this, we should fix that kaolin.render.mesh.rasterize soon too.
